great_expectations/data_context/data_context/cloud_data_context.py

```diff
@@ -86,6 +107,29 @@ def __init__(self):
         )
 
 
+class GXCloudConfigError(Exception):
+    def __init__(self, missing_keys: list[str]):
+        super().__init__(
+            "At least one of the following required keys are missing from the GX Cloud config: "
+            f"{missing_keys}"
+        )
+
+
+OPTIONAL_CLOUD_CONFIG_KEYS = [GXCloudEnvironmentVariable.WORKSPACE_ID]
+
+
+@dataclass
+class Workspace:
+    id: str
+    role: str
```

Comment:
Do we want to include is_default and workspace name? Otherwise might be harder to distinguish just by id for a user with multiple workspaces. This can be addressed later, if it becomes an issue.

Reply:
We can do this later. I think the rest endpoint will need to be updated to support that. I'm not to worried for now since user's need to set this and only switch between workspaces by getting a context. They can't currently list the workspaces since there is no workspace management functionality.